### PR TITLE
Tailwindformat

### PR DESCRIPTION
**Pull request by iLevyTate**
Updated format details

#263
#261 
#260 
#235 
#218

### CODE REVIEW

**Review by sanders41 — COMMENTED** [review]
Generally this looks good to me. The only thing I noticed is the days are overlapping each other on the home page. It looks to be most likely related to screen size.

image

**Comment by iLevyTate** [comment]
- You can visit http://localhost:3000/testing to preview the layout for the homepage. This is a test frontend page that I am currently using. https://github.com/xyvora/achievaim/pull/274/commits/b4cd7e2f427a90021b762453a919529615917008 

- I also fixed the overlapping in this commit: https://github.com/xyvora/achievaim/pull/274/commits/a5ad413dc79e81a02679d5ab8d6f2de42c7e8521

File: frontend/src/routes/testing/+page.svelte [hunk 1]
```diff
@@ -1,160 +1,432 @@
...
+                </div>
+              </figcaption>
+            </figure>
+          </div>
+        </div>
+        <!-- Card 2 -->
+        <div class="w-full bg-white rounded-xl p-4 shadow-md mb-4">
+          <!-- Content for Card 2 -->
+          <p>Card 2</p>
+        </div>
```
Comment (sanders41) [inline frontend/src/routes/testing/+page.svelte:224]: Is this supposed to be here?

File: frontend/src/routes/testing/+page.svelte [hunk 2]
```diff
@@ -1,160 +1,432 @@
...
-          </figcaption>
-        </figure>
+          </div>
+        </div>
+
+        <!-- Card 4 -->
+        <div class="w-full bg-white rounded-xl p-4 shadow-md mb-4">
+          <!-- Content for Card 4 -->
+          <p>Card 4</p>
+        </div>
```
Comment (sanders41) [inline frontend/src/routes/testing/+page.svelte:413]: Is this supposed to be here?

**Review by sanders41 — COMMENTED** [review]
If the plan is for the testing page to replace the home page go ahead and replace it here so we can test everything out in it's final format.

File: frontend/src/routes/testing/+page.svelte (hunk 1 above)
Comment (iLevyTate) [reply frontend/src/routes/testing/+page.svelte:224]: https://github.com/xyvora/achievaim/pull/274#discussion_r1285136122 → Yes and https://github.com/xyvora/achievaim/pull/274#discussion_r1285136257 → Yes.

I will create a pull request that will allow you to view the updated and final layout of the card page on the collapsecard branch. Before making any changes to the current home/main/display goal page, I wanted to work with you to ensure that I don't accidentally delete any logic or change anything you have already implemented. That's why I used format, design, and testing pages. To see the new drop-down collapsible card I was working on, please check out the testing and testingcomponent pages. The testing page shows an example layout with two "active" collapsible cards and one "completed" card, while the testingcomponent page displays a single collapsible card.

**Pull request merged**